### PR TITLE
Add static name and URL to non-CAN Scraper 

### PR DESCRIPTION
**Pull request by TomGoBravo**
This PR completes :crossed_fingers: https://trello.com/c/rI3EQlS6/976-add-nytimes-source-name-to-annotations

* Adds source name and URL to older data sources that are not part of the new CAN Scraper system

### CODE REVIEW

File: libs/datasets/sources/nytimes_dataset.py [hunk 1]
```diff
@@ -1,9 +1,12 @@
 from covidactnow.datapublic.common_fields import CommonFields
 from libs.datasets import data_source
+from libs.datasets.taglib import UrlStr
 
 
 class NYTimesDataset(data_source.DataSource):
     SOURCE_TYPE = "NYTimes"
+    SOURCE_NAME = "New York Times"
```
Comment (ghop02) [inline libs/datasets/sources/nytimes_dataset.py:8]: I think this should be "The New York Times” https://github.com/nytimes/covid-19-data#license-and-attribution

Comment (TomGoBravo) [reply libs/datasets/sources/nytimes_dataset.py]: Fixed in feb9b826

**Comment by TomGoBravo** [comment]
[Add NYTimes source name to annotations](https://trello.com/c/rI3EQlS6/976-add-nytimes-source-name-to-annotations)

**Comment by BrettBoval** [comment]
Those names look fine to me.

**Review by ghop02 — APPROVED** [review]

**Comment by mikelehen** [comment]
FWIW- I think cms_testing_dataset.py would ideally be in here too.

Name would be "The Centers for Medicare & Medicaid Services" and the URL would be https://data.cms.gov/stories/s/COVID-19-Nursing-Home-Data/bkwz-xpvg

**Pull request merged**